package-lock.json

```diff
@@ -0,0 +1,2658 @@
+{
```

Comment:
There weren't any dependency changes (since the soap dependency was removed). Does this package-lock.json match what gets generated when we run `yarn install` now?